library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608TrackRenderer.java

```diff
@@ -58,16 +67,40 @@
   private final Handler textRendererHandler;
   private final MediaFormatHolder formatHolder;
   private final SampleHolder sampleHolder;
-  private final StringBuilder captionStringBuilder;
+  private final SpannableStringBuilder captionStringBuilder;
   private final TreeSet<ClosedCaptionList> pendingCaptionLists;
 
   private boolean inputStreamEnded;
   private int captionMode;
   private int captionRowCount;
-  private String caption;
-  private String lastRenderedCaption;
+  private SpannableStringBuilder caption;
```

Comment:
Do you mean that the variable can be defined as the more generic supertype and I can still initialize it to SpannableStringBuilder? I guess it should work...